cli/loader/src/lib.rs

```diff
@@ -140,6 +140,8 @@ pub struct LanguageConfigurationJSON {
 #[derive(Serialize, Deserialize)]
 #[serde(rename_all = "kebab-case")]
 pub struct TreeSitterJSON {
+    #[serde(rename = "$schema")]
+    pub schema: String,
```

Comment:
Option so it doesn't break backwards compatibility, and we can omit from tests

Reply:
Ah thx good call with option. Also when making this a static str I get this error I am unsure how to resolve:

```
1. temporary value dropped while borrowed
   creates a temporary value which is freed while still in use [E0716]
2. argument requires that borrow lasts for `'static` [E0716]
```

In here:

```rust
    pub fn from_file(path: &Path) -> Result<Self> {
        Ok(serde_json::from_str(&fs::read_to_string(
            path.join("tree-sitter.json"),
        )?)?)
    }
```

Reply:
Ah yeah, String is fine then